Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/meteors.yml

```diff
@@ -15,7 +15,11 @@
       types:
         Blunt: 1
   - type: TimedDespawn
-    lifetime: 120
+    lifetime: 240
+  - type: PointLight # sunlight reflecting off them or something, prevents meteor stealthops sniping you
```

Comment:
Make the sprite unshaded instead.

Reply:
and yeah good idea

Reply:
ok unshaded meteors still have pretty low visibility so not sure about this
![image](https://github.com/user-attachments/assets/8745e0ea-85a1-4bd4-9916-89b0563a5043)

Reply:
![image](https://github.com/user-attachments/assets/aee52356-5705-4e8f-b20d-486f39cf26e9)
yes looks fine